src/creator.js

```diff
@@ -0,0 +1,1469 @@
+		if (groupChildren.length > 0) {
+			groupChildren.forEach((element) => {
+				element.scale.set(reloadScale.x, reloadScale.y, reloadScale.z)
+			})
+		}
+	}
+
+	// Function that creates the effect of an animation for the lines when moving the model.
+	function startMouseDownFire() {
+		reRenderLines()
+		mouseDownFireIntervals = setInterval(reRenderLines, 4) // 4 ms is the fastest setInterval can trigger.
+	}
+
+	// Arrow keys rotate the camera.
+	function arrowKeys(e) {
+		let radiantIncrement = (Math.PI * cameraRotationSpeed) / 180
+		switch (e.keyCode) {
+			case 37:
+				horizontalCameraRotation(radiantIncrement) // Left
+				break;
+
+			case 38:
+				verticalCameraRotation(radiantIncrement) // Up
+				break;
+
+			case 39:
+				horizontalCameraRotation(-radiantIncrement) // Right
+				break;
+
+			case 40:
+				verticalCameraRotation(-radiantIncrement) // Down
+				break;
+
+			case 18:
+				ctrlToggleState()
+				let modeIdBox = document.getElementsByClassName('optionsBox')[0]
+				if (modeIdBox.offsetHeight > 50) {
+					optionsBoxActions()
+				}
+				break;
+
+			default:
+				break;
+		}
+	}
+
+	function revertToPreviousState(e) {
+		switch (e.keyCode) {
+			case 18:
+				fireCTRL = true
+				addingLabelsBtnEffect()
+				break
+		}
+	}
+
+	function removeFromUI() {
+		document.querySelector('#image').remove()
+		document.querySelector('#imagewrapper').remove()
+		document.querySelector('#opacity-toggle').remove()
+		document.querySelector('#maincontrols').remove()
+		let helpBox = document.getElementById('help_moving')
+		for (let i = 7; i >= 3; i--) { helpBox.lastElementChild.remove() }
+	}
+
+	function createBtn(btnId, btnValue, btnParent) {
+		let btn = document.createElement('button')
+		btn.type = 'button'
+		btn.innerHTML = btnValue
+		btn.id = btnId
+
+		let controlNodeList = document.getElementById(btnParent)
+		controlNodeList.insertBefore(btn, controlNodeList.children[3])
+
+		return btn
+	}
+
+	function optionsBox() {
+		let modeIdBox = document.createElement('div')
+		modeIdBox.classList.add('optionsBox')
+		modeIdBox.innerHTML = 'Rotating'
+		modeIdBox.innerHTML += '<br>'
+
+		const controlNodeList = document.getElementById('controls')
+		controlNodeList.insertBefore(modeIdBox, controlNodeList.children[0])
+	}
+
+	function optionsBoxActions() {
+		let modeIdBox = document.getElementsByClassName('optionsBox')[0]
+		if (modeIdBox.offsetHeight < 50) {
+			modeIdBox.style.top = 0 + 'px'
+			modeIdBox.appendChild(mouseText())
+			modeIdBox.appendChild(keyboardText())
+
+		} else {
+			modeIdBox.style.top = ''
+			let childrenCount = modeIdBox.childElementCount - 1
+			console.log(childrenCount)
+			for (; childrenCount >= 0; childrenCount--) {
+				modeIdBox.children[childrenCount].remove()
+			}
+
+		}
+	}
+
+	function keyboardText() {
+		let keyboard = document.createElement('div')
+		keyboard.id = 'keyboard-text'
+		keyboard.classList.add('keyboardText')
+		keyboard.innerHTML = 'Arrow Keys:'
```

Comment:
We can condense this string to a single line, instead of concatenating multiple strings and using the `<br />` tag:

```
keyboard.innerHTML = 'Arrow Keys: \n &#183; Rotate camera. \n Option/alt: \n &#183; While pressing change between rotating and labeling'
```
You'll also have to add a new style in your CSS:
```
.keyboardText {
    white-space: pre;
}
```